gcloud/monitoring/timeseries.py

```diff
@@ -127,6 +127,7 @@ class Point(collections.namedtuple('Point', 'end_time start_time value')):
     :type start_time: string or None
     :param start_time: An optional start time in RFC3339 UTC "Zulu" format.
 
+    :type value: int
```

Comment:
From the description as "scalar" I'd imagine it's a float. Any clue from the unit tests?